libs/react-ui/src/lib/textarea/textarea.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, useState } from 'react';
```

Comment:
Not an big issue, but i noticed that the two existing React components differ in implementation. `Button` uses `class` syntax, and Textarea uses FC syntax. Would it make sense to use just one syntax everywhere?